Backend/app/rider-platform/rider-app/Main/src/Storage/Queries/Ride.hs

```diff
@@ -151,6 +152,7 @@ findStuckRideItems (Id merchantId) bookingIds now = do
     findAllWithDb
       [ Se.And
           [ Se.Is BeamB.providerId $ Se.Eq merchantId,
+            Se.Is BeamB.fareProductType $ Se.Not $ Se.Eq DFP.RENTAL,
```

Comment:
We don't want to sync rental ride ? 
Is it discussed with any PM ?

Reply:
Not discussed. But rental rides obviously can be more than 6 hours long, and we also can book a ride for future. So my opinion, it would be wrong to cancel rental bookings after 6 hours.